index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
якщо сайт на українскій мові
```suggestion
<html lang="uk">
```